src/scintillaeditor.cpp

```diff
@@ -273,9 +273,11 @@ void ScintillaEditor::applySettings()
 	if(Preferences::inst()->getValue("editor/enableAutocomplete").toBool())
 	{
 		qsci->setAutoCompletionSource(QsciScintilla::AcsAPIs);
-		qsci->setAutoCompletionFillupsEnabled(true);
+		qsci->setAutoCompletionFillupsEnabled(false);
+ 		qsci->setAutoCompletionFillups("(");
```

Comment:
Does this line matter if fillups are not enabled?

Reply:
I don't think, it will matter when fillups are enabled.